CMakeLists.txt

```diff
@@ -153,6 +157,17 @@ elseif(HTTPLIB_USE_BROTLI_IF_AVAILABLE)
 	set(HTTPLIB_IS_USING_BROTLI ${Brotli_FOUND})
 endif()
 
+if(HTTPLIB_REQUIRE_ZSTD)
+	find_package(ZSTD REQUIRED)
+	set(HTTPLIB_IS_USING_ZSTD TRUE)
+elseif(HTTPLIB_USE_ZSTD_IF_AVAILABLE)
+	find_package(ZSTD QUIET)
```

Comment:
find_package is case sensitive, so use `zstd` (lower-case) as that's how it's installed with its `zstdConfig.cmake` file.